post.html

```diff
@@ -0,0 +1,253 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="post.css">
+    <link href="https://fonts.cdnfonts.com/css/new-york-extra-large" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-mono" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-pro-display" rel="stylesheet">
+    <link href="https://fonts.cdnfonts.com/css/sf-pro-display" rel="stylesheet">
+
+    <title>Nordic Rose</title>
+</head>
+
+<body>
+    <header class="header">
+
+        <div class="top_padding_nordic">
+            <h1>
+                <picture>
+                    <source media="(max-width:854px)" srcset="photos\mobile_rose.png" />
```

Comment:
slash is still not fixed
```suggestion
                    <source media="(max-width:854px)" srcset="photos/mobile_rose.png" />
```